docs/_layouts/main.html

```diff
@@ -1,12 +1,11 @@
 ---
 layout: base
+extraJS:
+  - js/contributors.js
+extraCSS:
+  - css/content-contributors.css
```

Comment:
I think it is addressing your concerns @julienrf Now the script is loaded from static-site